pysportbot/authenticator.py

```
@@ -12,206 +9,292 @@
+                response.text[:200],
+            )
             raise ValueError(ErrorMessages.failed_login())
 
-        logger.info("Site login successful!")
+        try:
+            data = response.json()
+        except Exception as exc:
+            logger.error(
+                "JWT login returned non-JSON response. Body (truncated): %r",
+                response.text[:200],
+            )
```

Ruff (0.14.5)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)